src/components/Palette/index.css

```diff
@@ -1,3 +1,17 @@
+.palette {
+  background-image: linear-gradient(
+    152deg,
+    #f9f9f9 25%,
+    #ffffff 25%,
+    #ffffff 50%,
+    #f9f9f9 50%,
+    #f9f9f9 75%,
+    #ffffff 75%,
+    #ffffff 100%
+  );
+  background-size: 102.24px 54.36px;
```

Comment:
The size seems a bit too strict, otherwise 👍

Reply:
The only way I know how to do these striped gradients are with some very specific `background-size` values. Useful to note that this doesn't restrict the element to that size.

Feel free to suggest an alternative if you've got one